src/main.cpp

```diff
@@ -298,7 +298,9 @@ int main(int argc, char *argv[])
     core.display  = display;
     core.ev_loop  = wl_display_get_event_loop(core.display);
     core.backend  = wlr_backend_autocreate(core.display);
-    core.renderer = wlr_backend_get_renderer(core.backend);
+    core.renderer = wlr_renderer_autocreate(core.backend);
```

Comment:
Can we directly create a GLES renderer here? That's the only thing we support.

Reply:
Looks like it is possible, `wlr_gles2_renderer_create_with_drm_fd` is public, I'll try that now